ci/local/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DOCKER_IMAGE="gpuci/rapidsai:cuda10.0-ubuntu16.04-gcc5-py3.6"
+DOCKER_IMAGE="gpuci/rapidsai:0.15-cuda10.1-devel-ubuntu16.04-py3.6"
```

Comment:
Can we programmatically get the 0.15 from the git tag or something similar? Otherwise we'll have to update this every release.